src/main.rs

```diff
@@ -87,9 +95,13 @@ fn main() {
         });
 
         if let Err(e) = res {
-            println!("An error occured: {}", e);
+            println!("An error occured: {}\nresting {} Seconds", e, backoff_seconds);
```

Comment:
I think he means *resting* with regard to `sleep_ms`. But maybe *sleeping* would be less confusing. Or *rusting* :wink: